library/stdio/openiob.c

```diff
@@ -93,7 +93,15 @@ __open_iob(struct _clib4 *__clib4, const char *filename, const char *mode, int f
     SHOWMSG("allocating file buffer");
 
     /* Allocate a little more memory than necessary. */
-    buffer = AllocVecTags(BUFSIZ + (__clib4->__cache_line_size - 1), AVT_Type, MEMF_SHARED, AVT_ClearWithValue, AVT_Alignment, __clib4->__cache_line_size, 0, TAG_DONE);
```

Comment:
This allocVecTags definition was wrong, with arguments in disorder (AVT_ClearWithValue followed by AVT_Alignment)